gabbi/case.py

```diff
@@ -27,7 +27,9 @@
 import re
 import sys
 
+import wsgi_intercept
```

Comment:
That's a habit from openstack land where one is obliged to only import modules. I'll adjust it to non openstack styles